lib/diagnostic.rb

```diff
@@ -131,6 +168,10 @@ def kick
     @damage *= multiplier
   end
 
+  def get_possible_moves
+    puts "kick, move, punch"
+  end
+
```

Comment:
The way to make the method chaining work is for `punch`, `kick`, and `move` to return `self`.That way, `ComboAttack.new.punch.move('left').kick.damage` is equivalent to

```ruby
combo = ComboAttack.new
combo.punch
combo.move('left')
combo.kick
combo.damage
```